### ISSUE TITLE
MessageBox:  A11y issue that Title should be H2 instead of H5

### ISSUE THREAD

**Issue by mydavidzhang** [SAP/ui5-webcomponents-react#1453]
**Describe the bug**
There is A11y issue that MessgeBox Title should be H2 instead of H5

**Isolated Example**
https://github.com/SAP/ui5-webcomponents-react/blob/main/packages/main/src/components/MessageBox/index.tsx#L200
<header className={classes.header} data-type={type}>
  {iconToRender}
  <Title level={TitleLevel.H5}>{titleToRender()}</Title>
</header>

**To Reproduce**
Render MessageBox
Look at header Title
Title is H5

**Expected behavior**
Header Title should be H2

**Screenshots**
![warning heading](https://user-images.githubusercontent.com/10460331/112782233-65aaee00-9001-11eb-8a6c-b6803f321a02.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 0.29.11
`@ui5/webcomponents-react` version: 0.129.8
Operating System:
Browser: Chrome

**Additional context**
For Successfactors

**Comment by MarcusNotheis** [comment]
Released in `0.129.10` and `0.131.3`.
